changelogs/unreleased/6779-discovery-resources-allow-matching-by-id.yml

```
@@ -0,0 +1,8 @@
+description: >
+  Add filtering capabilities on DiscoveredResourceView. Discovered resource can now be filtered based on
+  whether they are already managed or not.
```

The word DiscoveredResourceView doesn't mean much to an end user.
```suggestion
  Add support to the `GET /api/v2/discovered` endpoint to filter the discovered resources on whether they are managed or not.
```